src/scripts/scripts/zone/caverns_of_time/old_hillsbrad/boss_leutenant_drake.cpp

```diff
@@ -50,17 +50,21 @@ bool GOUse_go_barrel_old_hillsbrad(Player *player, GameObject* go)
 ## boss_lieutenant_drake
 ######*/
 
-#define SAY_AGGRO                  -1560007
-#define SAY_SLAY1                  -1560008
-#define SAY_SLAY2                  -1560009
-#define SAY_MORTAL                 -1560010
-#define SAY_SHOUT                  -1560011
-#define SAY_DEATH                  -1560012
-
-#define SPELL_WHIRLWIND            31909
-#define SPELL_HAMSTRING            9080
-#define SPELL_MORTAL_STRIKE        31911
-#define SPELL_FRIGHTENING_SHOUT    33789
+enum LieutenantDrake
+{
+    SAY_AGGRO                 = -1560007,
+    SAY_SLAY1                 = -1560008,
+    SAY_SLAY2                 = -1560009,
+    SAY_MORTAL                = -1560010,
+    SAY_SHOUT                 = -1560011,
+    SAY_DEATH                 = -1560012,
+
+    SPELL_WHIRLWIND           = 31909,
+    SPELL_HAMSTRING           = 9080,
+    SPELL_MORTAL_STRIKE       = 31911,
+    SPELL_FRIGHTENING_SHOUT   = 33789,
+    SPELL_EXPLODING_SHOT      = 33792
```

Comment:
http://en.cppreference.com/w/cpp/language/enum

The last entry within an enum ddoes not receive a comma, else the compiler would expect something to follow.